jackson/src/main/resources/reference.conf

```diff
@@ -1,40 +1,25 @@
 play.modules.enabled += com.lightbend.lagom.javadsl.jackson.JacksonModule
 
-lagom.serialization.json {
-  
-  #//#jackson-modules
-  # The Jackson JSON serializer will register these modules.
-  # It is also possible to use jackson-modules = ["*"] to dynamically
-  # find and register all modules in the classpath.  
-  jackson-modules += "com.fasterxml.jackson.module.paramnames.ParameterNamesModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.jdk8.Jdk8Module"
-  jackson-modules += "com.fasterxml.jackson.datatype.jsr310.JavaTimeModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.pcollections.PCollectionsModule"
-  jackson-modules += "com.fasterxml.jackson.datatype.guava.GuavaModule"
-  #//#jackson-modules  
-  
-  # The serializer will compress the payload if the message class
-  # implements the CompressedJsonable marker interface and the payload 
-  # is larger than this value. Only used for remote messages within 
-  # the cluster of the service.
-  compress-larger-than = 1024b
-  
-  # Define data migration transformations of old formats to current 
-  # format here as a mapping between the (old) class name to be
-  # transformed to the JacksonJsonMigration class that implements
-  # the transformation.  
-  migrations {
-  }
-}
-
 akka.actor {
   serializers {
-    lagom-json = "com.lightbend.lagom.internal.jackson.JacksonJsonSerializer"
+    # Placeholder for Lagom 1.5.x JacksonSerializer.
+    # Needed for deserialization of messages during rolling update, and old data (events, snapshots)
+    # that were serialized with the Lagom 1.5.x JacksonSerializer.
+    # Note that by default there is no
+    old-lagom-json = "com.lightbend.lagom.internal.jackson.OldJacksonJsonSerializer"
   }
   serialization-bindings {
-    "com.lightbend.lagom.serialization.Jsonable" = lagom-json
+    # Bind Jsonable to Akka JacksonJsonSerializer
+    "com.lightbend.lagom.serialization.Jsonable" = jackson-json
   }
   serialization-identifiers {
-    "com.lightbend.lagom.internal.jackson.JacksonJsonSerializer" = 1000002
+    # For Lagom 1.5.x JacksonSerializer.
+    "com.lightbend.lagom.internal.jackson.OldJacksonJsonSerializer" = 1000002
   }
```

Comment:
My thinking about how to support rolling update:

* Two step roll out
* First step would be to use `"com.lightbend.lagom.serialization.Jsonable" = old-lagom-json` as the serialization-bindings
  * that means that nothing will be serialized (toBinary) with Akka's serializer, but still use the old serializer (with id 1000002)
  * purpose of first step is to add Akka JacksonJsonSerializer in classpath and have it prepared for deserialization in old nodes in next step
* Second step is to roll out `"com.lightbend.lagom.serialization.Jsonable" = jackson-json` (which is also the default config)
  * this means that messages and data in new nodes will start using Akka's JacksonJsonSerializer for serialization (toBinary), and old nodes can deserialize (fromBinary) that since it was enabled in previous step
  * old messages/data can still be deserialized because the `old-lagom-json` is still registered to handle id 1000002

Reply:
I don't the ActorRef serializer is a strong reason for that. We can easily backport it to Lagom 1.5.x.

However, we have to think about how the migration with a rolling update would look like for this. One challenge is the `sender` vs the `replyTo: ActorRef` in the messages. I added that task as part of https://github.com/akka/akka/issues/26718#issuecomment-498053886